src/deluge/model/mod_controllable/mod_controllable_audio.h

```diff
@@ -159,7 +162,7 @@ class ModControllableAudio : public ModControllable {
 	StereoSample grabbedSample;
 	StereoSample lastGrabbedSample;
 
-	SideChain compressor; // Song doesn't use this, despite extending this class
+	SideChain sidechain; // Song doesn't use this, despite extending this class
```

Comment:
Similarly I think `Song` does use this now?

Reply:
I was confusing this with the compressor, which I'm pretty sure the song does use.